src/methods.jl

```diff
@@ -30,3 +30,33 @@ function solvehook(m::Model; method::MultiMethod = WeightedSum(),
     status = multisolve(m, data, method, linearity)
     return status
 end
+
+
+"""
+Extract the Utopia point from the MultiData struct.
+"""
+function getutopia(m::MultiData)
+    Fmin = diag(m.Phi)
+
+    # Convert values depending on whether
+    # we maximize or minimize them.
+    sensemap = Dict(:Min => 1.0, :Max => -1.0)
```

Comment:
building a dictionary for this seems like a lot of overhead. The suggestion below also avoids building one intermediate array
```suggestion
multiplier = (obj.sense == :Max ? -1. : 1.] for obj in m.objectives)
return [Fmin[i] * mlt for (i,mlt) in enumerate(multiplier)]
```